test/integration/utils.ts

```diff
@@ -1,9 +1,60 @@
-import { Client } from "../../src";
-import { SignedTransaction } from "../../src/common/types/objects";
+import { Client } from "xrpl-local";
+import {
+  FormattedOrderSpecification,
+  SignedTransaction,
+} from "xrpl-local/common/types/objects";
+
+import { walletAddress } from "./wallet";
 
 const masterAccount = "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh";
 const masterSecret = "snoPBrXtMeMyMHUVTgbuqAfg1SUTb";
 
+export async function makeTrustLine(
+  testcase: Mocha.Context,
+  address: string,
+  secret: string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- The return type is legitimately any
+): Promise<any> {
```

Comment:
I think these should be a `Promise<void>`, since `.then(async () => ledgerAccept(client));` returns void.